### PR TITLE
Source Facebook-Marketing: updated facebook_business==16.0.0

### LINKED ISSUE #22549: Source Facebook Marketing -  failing asking for upgrade to v16
## Environment
- **Airbyte version**:  0.40.32
- **OS Version / Instance**: GCP n2
- **Deployment**: Docker
- **Source Connector and version**: Facebook Marketing 0.2.84
- **Destination Connector and version**: 
- **Step where error happened**: Setup new connection

## Current Behavior
The connection tests failed, with an error message "
```
The connection tests failed.
FacebookAPIException('Error: 2635, (#2635) Your app has been upgraded to version v16.0, please use this version or newer. This can be verified in the settings tab on the App Dashboard.')
```

It is no longer possible to create Facebook Apps with v15.0 (that Facebook Marketing 0.2.84 supports). It is also not possible to downgrade new Facebook Apps from v16.0 to v15.0.

## Expected Behavior
The connection tests should succeed

## Logs
```
  Message: Call was not successful
  Method:  GET
  Path:    https://graph.facebook.com/v15.0/act_<id has been removed by ticket poster>/
  Params:  {}
  Status:  400
  Response:
    {
      "error": {
        "message": "(#2635) Your app has been upgraded to version v16.0, please use this version or newer. This can be verified in the settings tab on the App Dashboard.",
        "type": "OAuthException",
        "code": 2635,
        "fbtrace_id": "<id has been removed by ticket poster>"
      }
    })
```

## Steps to Reproduce
1. Create a connector with Facebook Marketing
2. Test the connection

## Are you willing to submit a PR?
No

For additional details, please refer to to https://github.com/airbytehq/airbyte/issues/18167, which is a ticket for the last version update.

Image showing the result of a connection test
image

Image showing that it is no longer possible to select v.15.0 as the version in newly created Facebook Apps
image

### PR DESCRIPTION
**Pull request by darynaishchenko**
on-call https://github.com/airbytehq/airbyte/issues/22549

### CODE REVIEW

**Review by bazarnov — APPROVED** [review]

File: airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml [hunk 1]
```diff
@@ -53,3 +53,4 @@ acceptance_tests:
   full_refresh:
     tests:
       - config_path: "secrets/config.json"
+        timeout_seconds: 2400
```
Comment (maxi297) [inline airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml:56]: Out of curiosity, why was this added?

**Review by maxi297 — APPROVED** [review]
LGTM!

File: airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml (hunk 1 above)
Comment (darynaishchenko) [reply airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml:56]: sometimes it fails with timeout

**Comment by HardikNarendra** [comment]
Is the version problem solved? how much time will it take to merge the code?

**Review by lazebnyi — APPROVED** [review]

**Comment by kaokei** [comment]
I thought this issue has been resolved now. 
But I downloaded the repo, and exec this command `./run-ab-platform.sh`. It still use facebook api v15 other than v16 when I create facebook market source.
I don't know why. could anyone help me?
